src/components/grid-aware/HomePageLargeParagraph/HomePageLargeParagraph.module.css

```diff
@@ -0,0 +1,59 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  justify-items: center;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-bottom: 120px;
+  padding-top: 120px;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+  text-align: center;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: repeat(2, auto);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+  }
+
+  .description {
+    margin-top: 40px;
+  }
```

Comment:
It's a bit weird to have the media-query version of this selector defined higher than the non-media-query version. Everywhere else in our codebase, we always define the default styles first and then the media query overrides later.

Actually, now that I look at it more closely, this is redundant because there should be a 40px top margin in both the mobile and desktop views, so there's no need to override this in a media query.

Reply:
👍 Good to know that the media-query version should be lower than the non-media query version. 

Deleted the media query for description because as you said, the 40px top margin applies to both mobile and desktop views without the need to override with a media query.